client/screens/MapScreen.js

```diff
@@ -21,19 +40,52 @@ const styles = {
 export default class MapScreen extends React.Component {
   state = {
     region: null,
-    pins: []
+    pins: [],
+    location: null,
+    errorMessage: null
+  }
+
+  componentWillMount() {
```

Comment:
Component will mount is deprecated. Use componentDidMount*

Reply:
This might be a comment for @el-fer